Sources/AppStoreConnectCLI/Commands/Certificates/ListDownloadCertificatesCommand.swift

```diff
@@ -70,19 +56,19 @@ struct ListDownloadCertificates: CommonParsableCommand {
         if let downloadPath = downloadPath {
             try certificates.forEach { (certificate: Certificate) in
                 guard let content = certificate.content else {
-                    throw CommandError.invalidContent
+                    throw CertificatesError.invalidContent
                 }
 
                 do {
-                    let folder = try Folder(path: downloadPath)
-
-                    let file = try folder.createFile(
+                    let file = try File.createFile(
```

Comment:
🤔 What sort of error does this throw? Could we just use that error rather than mapping to `CertificatesError.invalidPath`?